packages/shared/src/components/tabs/TabContainer.tsx

```diff
@@ -15,6 +16,7 @@ export interface TabProps<T extends string> {
   className?: string;
   style?: CSSProperties;
   showHeader?: boolean;
+  url?: string;
```

Comment:
in its current form, it will only work with pathname, since we are using `router.push`, it won't work with an absolute path. would you prefer this prop be named `pathname` instead?

Reply:
`router.push` can work with absolute urls as well, I was just curious of the usage and if we want it, up to you on the naming 👍

Reply:
I guess I need to keep up to date with things more, last time I checked, using `router.push` with a URL that is outside the app domain was not an option 😬 let's leave it as `url` for now then if no one else has a strong preference